### PR TITLE
Add small railroad wheel

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: Content "Add small railroad wheel"```

#### Purpose of change
Follow up on #34995

#### Describe the solution
Adds a pair of small railroad wheels and a recipe for it. It is copied from the regular railroad wheel and I changed some things. You can install it on a bicycle for example and ride it on the ground instead of rails but I did not add a steerable wheel so it's impossible to turn if you're not on the rails. If you are on rails, the vehicle turns according to the rail layout

#### Describe alternatives you've considered
I remember the Discord discussion and the idea of wooden wheels lined with steel came up but in my opinion these steel wheels are a better fit but I'm open for discussion.

#### Testing
The item:
image

The recipe:
image

The part:
image

#### Additional context
N/A

### CODE REVIEW

File: data/json/recipes/recipe_vehicle.json [hunk 1]
```diff
@@ -537,6 +537,20 @@
...
+    "result": "wheel_rail_small_pair",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "skills_required": [ "mechanics", 1 ],
+    "difficulty": 3,
+    "time": "50 m",
+    "autolearn": true,
+    "using": [ [ "blacksmithing_standard", 5 ], [ "steel_standard", 12 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 1 } ],
```
Comment (Rail-Runner) [inline data/json/recipes/recipe_vehicle.json]: HAMMER quality is unnecessary here; already included into "blacksmithing_standard" requirement.

Comment (snipercup) [reply data/json/recipes/recipe_vehicle.json]: Thanks, I removed it.

File: data/json/items/vehicle/wheel.json [hunk 2]
```diff
@@ -186,6 +186,19 @@
     "diameter": 20,
     "width": 4
   },
+  {
+    "id": "wheel_rail_small_pair",
+    "name": "pair of small rail wheels",
+    "type": "WHEEL",
+    "description": "A pair of small rail wheels.  It fits small railroad rails found in places like amusement parks and the backyard of people with lots of time on their hands. Mostly for hobby purposes rather then transportation.",
```
Comment (kevingranade) [inline data/json/items/vehicle/wheel.json]:
```suggestion
    "description": "A pair of small rail wheels.  It fits small railroad rails found in places like amusement parks.  Mostly for hobby purposes rather then transportation.",
```

File: data/json/vehicleparts/wheel.json [hunk 3]
```diff
@@ -131,6 +131,20 @@
...
+  {
+    "id": "rail_wheel_small_pair",
+    "type": "vehicle_part",
+    "name": "pair of small rail wheels",
+    "item": "wheel_rail_small_pair",
+    "copy-from": "rail_wheel",
+    "symbol": "=",
+    "difficulty": 2,
+    "durability": 150,
+    "description": "A pair of small rail wheels.  It balances the vehicle weight on the double small rail tracks it's supposed to ride on.  It perform terribly when not on a rail.  Moreover, you can't steer it.",
```
Comment (kevingranade) [inline data/json/vehicleparts/wheel.json]:
```suggestion
    "description": "A pair of small rail wheels.  It performs well when riding on a pair of small rails.  It performs terribly when not on a rail.  Moreover, you can't steer it.",
```

**Pull request merged**